vcenter/vcenter.py

```diff
@@ -94,58 +134,64 @@ def deploy(self, template, machine_name, **kwargs):
 
             raise e
 
-        objView = self.content.viewManager.CreateContainerView(self.content.rootFolder,
-                                                               [vim.VirtualMachine],
-                                                               True)
+        template = self.__search_machine_by_name(template)
```

Comment:
Reusing `template` variable here is no good, as the meaning on the right side of the expression is different than on the left side of the expression.

Reply:
I would suggest to change `template` parameter to `template_name`.

Then line 137 becomes:
```
template = self.__search_machine_by_name(template_name)
```

Then line 140 could become:
```
raise RuntimeError("Template {} hasn't been found".format(template_name))
```